website/blog/2020-08-31-Hydra_1.0.md

```diff
@@ -0,0 +1,431 @@
+  dst:
+    host: localhost
+    port: 3306
+```
+</div></div>
+
+Learn more at [Advanced/Overriding packages](/docs/next/advanced/overriding_packages).
+
+## Command line grammar (Overrides grammar)
+Hydra 1.0 introduces a proper parser and grammar for the command line syntax, supporting a multitude of new use cases.
+
+A short list of examples:
+
+
+<div className="row">
+<div className="col col--6">
+
+```text
+Override as usual
+
+
+
+```
+
+```text
+Prefix with + to add a new field to
+the config
+
+
+
+```
+
+```text
+Prefix with ~ to delete a field from
+the config
+
+```
+
+```text
+Cast values to coerce their type.
+
+
+
+```
+
+```text
+Support for dictionaries and lists
+
+
+
+
+
+
+
+```
+
+</div><div className="col  col--6">
+
+```yaml
+$ python my_app.py db.user=root
+db:
+  user: root
+  pass: secret
+```
+
+```yaml
+$ python my_app.py +db.timeout=10
+db:
+  user: omry
+  pass: secret
+  timeout: 10
+```
+
+```yaml
+$ python my_app.py '~db.pass'
+db:
+  user: omry
+```
+
+```yaml
+$ python my_app.py 'db.pass=str(42)'
+db:
+  user: omry
+  pass: '42'
+```
+
+```yaml
+$ python my_app.py \
+  '+db.params={a:10,b:20}'
+db:
+  user: omry
+  pass: secret
+  params:
+    a: 10
+    b: 20
+```
+
+</div></div>
+
+Learn more at [Basic](/docs/next/advanced/override_grammar/basic) and [Extended Override grammar](/docs/next/advanced/override_grammar/extended) pages. 
+
+## Sweeper improvements
+The [Extended Override grammar](/docs/next/advanced/override_grammar/extended) is making all sweepers more powerful.
+
+
+<div className="row">
+<div className="col col--6">
+
+```text
+Use glob to easily select
+multiple config group options
+```
+
+```text
+Use range to specify a range of 
+numbers
+
+
+```
+
+```text
+You can sort sweep to run order
+
+
+
+
+
+
+```
+
+```text
+You can also shuffle sweeps to run
+in random order
+
+```
+
+
+</div><div className="col  col--6">
+
+```python
+$ python my_app.py -m 'db=glob(*)'
+# Will sweep over all db options
+```
+
+```python
+$ python my_app.py --multirun \
+  '+number=range(1,4)'
+# number will take the values
+# 1,2 and 3 
+```
+
+
+```python
+$ python my_app.py --multirun \
+  '+num=sort(3,1,2)'
+# Sweep over num in order
+
+$ python my_app.py --multirun \
+  '+num=sort(3,1,2,reverse=true)'
+# Sweep over num in reverse order
+```
+
+```python
+$ python my_app.py --multirun \
+  '+num=sort(3,1,2,reverse=true)'
+# Sweep over num in random order
+```
+
+
+
+</div></div>
+
+
+## Compose API improvements
+The experimental Compose API is maturing. It is now possible to initialize Hydra in one of 3 ways:
+- `initialize()`: Initialize with a config path relative to the caller
+- `initialize_config_module()` : Initialize with config_module (absolute)
+- `initialize_config_dir()` : Initialize with a config_dir on the file system (absolute)
+
+All initialization methods can be used to initialize Hydra globally or as a context.  
+Learn more at [Experimental/Compose API](/docs/next/experimental/compose_api).
```

Comment:
Just highlighting "Experimental/Compose API". Maybe you want it to be "Compose API"